dnsmasq.template

```diff
@@ -0,0 +1,25 @@
+# Template file for dnsmasq generation
```

Comment:
I like the idea of putting a sample template in the repo!! We should, however call it like `dnsmasq.template.sample` or something since this clearly has site-specific data in it. 

I should do this for radvd instead of putting it in the docs (if you wanna do that in this PR, feel free, but you don't need to all).

Reply:
Done. Won't be touching radvd.